src/current/v24.2/log-sql-statistics-to-datadog.md

```diff
@@ -88,12 +88,9 @@ Set the [`sql.telemetry.query_sampling.mode` cluster setting]({% link {{ page.ve
 SET CLUSTER SETTING sql.telemetry.query_sampling.mode = 'statement';
 ~~~
 
-Set the [`sql.telemetry.query_sampling.max_event_frequency` cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-telemetry-query-sampling-max-event-frequency) to `100000` to emit query events at a higher rate per second than the default value of `8`, which is extremely conservative for the Datadog HTTP API. This cluster setting controls the max event frequency at which CockroachDB samples queries for telemetry.
+Configure the following [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) to a value that is dependent on the level of granularity you require and how much performance impact from frequent logging you can tolerate:
 
-{% include_cached copy-clipboard.html %}
-~~~ sql
-SET CLUSTER SETTING sql.telemetry.query_sampling.max_event_frequency = 100000;
-~~~
+- [`sql.telemetry.query_sampling.max_event_frequency`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-telemetry-query-sampling-max-event-frequency) (default `8`) is the max event frequency (events per second) at which we sample executed queries for telemetry. If sampling mode is set to `'transaction'`, this setting is ignored. In practice, this means that we only sample an executed query if 1/max_event_frequency seconds have elapsed since the last executed query was sampled. Sampling impacts the volume of query events emitted which can have downstream impact to workload performance and third-party processing costs. Slowly increase this sampling threshold and monitor potential impact.
```

Comment:
same as above re: formatting of `max_event_frequency` inline with the prose, and duplicating the default value info here vs. cluster settings page

Reply:
- updated formatting of `max_event_frequency`.
- keeping default values because they provide magnitude of initial setting